JS/oop-assignment-forum/Comment.js

```diff
@@ -0,0 +1,7 @@
+import TextAddition from "./TextAddition.js"
+
+export default class Comment extends TextAddition{
+    constructor({text}){
```

Comment:
here you just overwrite the constructor then call it again. Can just leave it as `class Comment extends TextAddition`